docs-conceptual/azureadps-2.0/migration-faq.yml

```diff
@@ -0,0 +1,40 @@
+          
+          Azure AD PowerShell will continue to function after June 2022 to allow users more time to migrate to [Microsoft Graph PowerShell](https://docs.microsoft.com/en-us/powershell/microsoftgraph/overview). This will allow smooth customer migration to Microsoft Graph PowerShell SDK with a graceful shutdown of the Azure AD Graph endpoint. We encourage you to migrate to the Microsoft Graph PowerShell SDK. 
+          
+
+      - question: What are we doing about the gaps in functionality between Azure AD and Microsoft Graph PowerShell modules?
+        answer: |
+          As part of the parity work, we're looking at the MSOnline (MSOL) and Azure AD PowerShell glossary of cmdlets and mapping the functionality to Microsoft Graph PowerShell. This is to make sure that all existing scenarios are available in Microsoft Graph PowerShell. We started working on [a cmdlet map](https://docs.microsoft.com/en-us/powershell/microsoftgraph/azuread-msoline-cmdlet-map) for both MSOnline and Azure AD PowerShell cmdlets to Microsoft Graph PowerShell as part of the migration documentation. 
+          
+          Users have also reached out with specific scenarios and cmdlets that we'll add to Microsoft Graph PowerShell. We're also working on tools such as the [Find-MgGraphCommand](https://docs.microsoft.com/en-us/powershell/microsoftgraph/find-mg-graph-command) and [Find-MgGraphPermission](https://docs.microsoft.com/en-us/powershell/microsoftgraph/find-mg-graph-permission) that will assist users in discovering and using the appropriate Microsoft Graph PowerShell cmdlets.  
+
+      - question: What are we doing to improve the Microsoft Graph PowerShell user experience?
+        answer: |
+          Microsoft Graph PowerShell cmdlets are autogenerated from the Microsoft Graph API schema. The process has its advantages, for example, providing a rich set of cmdlets that cover the whole API landscape. The disadvantage is that the resultant cmdlets may not be user-friendly compared to human authored cmdlets. Consequently, we're actively working on usability improvements.  
+          
+          The Microsoft Graph PowerShell SDK is also open source and users can contribute to the usability work or add new scenarios to manage Azure AD.
+          
+      - question: How are we going to support users who are migrating to Microsoft Graph PowerShell?
+        answer: |
+          We're working on additional tools that will ease the migration of existing scripts, that target Azure AD PowerShell, so that they can use Microsoft Graph PowerShell SDK instead.
```

Comment:
```suggestion
          We're working on additional tools that will help migrate existing scripts that target Azure AD PowerShell so that these scripts can use Microsoft Graph PowerShell cmdlets instead.
```

Reply:
Or drop cmdlets altogether and just say "Microsoft Graph PowerShell" but leave out the SDK.